static/app/components/hovercard.tsx

```diff
@@ -127,10 +127,12 @@ function Hovercard(props: HovercardProps): React.ReactElement {
   // If show is not set, then visibility state is uncontrolled
   const isVisible = props.show === undefined ? visible : props.show;
 
-  const hoverProps = useMemo((): Pick<
-    React.HTMLProps<HTMLDivElement>,
-    'onMouseEnter' | 'onMouseLeave'
-  > => {
+  const hoverProps = useMemo(():
+    | {
+        onMouseEnter: React.MouseEventHandler<HTMLDivElement>;
+        onMouseLeave: React.MouseEventHandler<HTMLDivElement>;
+      }
+    | {} => {
```

Comment:
Why not just `Pick` seems clearer to me?

Reply:
Pick achieves the same, but it's one level higher in the abstraction. Imo better to just pull the type directly so it's less ambiguous what Pick<HTMLAttributes<Element>["onClick"]>> is